frontend/package.json

```diff
@@ -219,6 +219,7 @@
     "text-encoding": "0.x",
     "typesafe-actions": "^4.2.1",
     "url-search-params-polyfill": "2.x",
+    "victory-core": "^36.2.1",
```

Comment:
Lets add a **note** or a **ToDo** to remove once https://issues.redhat.com/browse/OCPBUGS-4279 is fixed